src/volta-shim.rs

```diff
@@ -1,35 +1,12 @@
-mod common;
+use std::process::{Command, ExitCode};
 
-use common::{ensure_layout, Error, IntoResult};
-use volta_core::error::{report_error, ExitCode};
-use volta_core::log::{LogContext, LogVerbosity, Logger};
-use volta_core::run::execute_shim;
-use volta_core::session::{ActivityKind, Session};
-use volta_core::signal::setup_signal_handler;
-
-pub fn main() {
-    Logger::init(LogContext::Shim, LogVerbosity::Default)
-        .expect("Only a single Logger should be initialized");
-    setup_signal_handler();
-
-    let mut session = Session::init();
-    session.add_event_start(ActivityKind::Tool);
-
-    let result = ensure_layout().and_then(|()| execute_shim(&mut session).into_result());
+pub fn main() -> ExitCode {
+    let result = Command::new("volta-runner").args(std::env::args()).status();
```

Comment:
Hi Charles!

Thank you for the feedback.

So you prefer this change is changed only for Windows, right? We can make this condition, no problem.

However, I'm not sure if the `volta run`  proposal would solve the issues. The main use case for the file symlink is the global installs. I made a test with `volta run npm install -g ng` and `volta run ng` and it didn't work. I don't know if I misunderstood something.

Reply:
Yeah, if we move forward here, we should limit the scope to only for Windows, so that we have the smallest possible blast radius.

For the `volta run` idea, I want to do a quick prototype to see if it's feasible, then we can look at which approach makes the most sense.

And thank you again for persisting with this change, despite our busy schedules making the lead times really long! It will be a big improvement in Windows behavior when we don't need the "Developer Mode" warning 💥

Reply:
Great! So I'll wait for you before making both changes.